templates/taxi/car_detail.html

```diff
@@ -12,8 +12,23 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
+  <div style="display: flex; justify-content: space-between; align-items: center;">
+    <h1>Drivers</h1>
+    <div>
+        {% if request.user in object.drivers.all %}
```

Comment:
The use of `object` here should be replaced with `car` to maintain consistency with the rest of the template. This ensures that the template logic correctly references the `car` context variable.